grammar.js

```diff
@@ -239,14 +239,13 @@ module.exports = grammar({
       optional($._automatic_semicolon)
     )),
 
+    else_branch: $ => seq('else', $._statement),
```

Comment:
Could you change this name to `else_clause`? I think that for these parts of statements, we usually use the term "clause" in this and other tree-sitter grammars. See for example `catch_clause`, `finally_clause` in this file.

Reply:
changed to `else_clause`

https://github.com/tree-sitter/tree-sitter-javascript/pull/143/commits/e03246ea76583c010b327a62e9f34168fccb9f73